internal/config/default.yaml

```diff
@@ -5,5 +5,9 @@ MongoDB:
   host: localhost
   name: things_db
 
+Cloud:
```

Comment:
Why did you choose `cloud` instead of `users`?

Reply:
Makes sense! :smile: 
Done!